src/AppRoot.tsx

```diff
@@ -103,6 +112,25 @@ const AppRoot: React.FC = () => {
     [navigate, location]
   );
 
+  const plugins: Plugin[] = [
```

Comment:
We could memoize it without dependencies (`useMemo(() => [{name: 'ai-insights'/*...*/}], [])`)